{"level":"ERROR","location":"[get_call:143] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2024-10-26 04:29:54,849+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:143] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2024-10-26 04:29:55,127+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2531c1eb0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:25:45,987+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e251d21a30>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:25:45,989+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2531f2d50>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:25:56,491+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2531c3170>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:25:56,492+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2527a24b0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:27:39,351+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2527d46e0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:27:39,352+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2527a0e60>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:58:41,399+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2531f3650>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:58:41,400+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2527a35f0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:59:49,434+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2527d4800>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 06:59:49,435+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e2531f1fa0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 07:44:02,319+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x76e25278dcd0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 07:44:02,320+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee7540170>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:00,535+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee7f46f30>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:00,536+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee75b47a0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:12,324+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee75b4230>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:12,325+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee75b5d60>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:20,540+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x79eee7513380>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 08:28:20,541+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x726127f73b30>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:34:51,308+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x726127f71250>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:34:51,309+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x726127f70710>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:35:56,301+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x726127f70620>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:35:56,302+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x72612c0e2390>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:35:59,669+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x726127504dd0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-26 09:35:59,671+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:143] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2024-10-27 13:19:09,560+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:143] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2024-10-27 13:19:09,728+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"},"timestamp":"2024-10-27 14:39:10,298+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"},"timestamp":"2024-10-27 14:39:14,424+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"},"timestamp":"2024-10-27 14:40:08,381+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ec9d11f6450>: Failed to establish a new connection: [Errno 113] No route to host'))"},"timestamp":"2024-10-27 14:55:51,699+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11f7bf0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:55:51,700+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11160c0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:11,702+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d1102630>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:11,704+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11f6e10>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:31,706+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11463f0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:31,707+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11474a0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:51,709+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11abfe0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:56:51,710+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d1147260>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:57:11,713+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11f76b0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:57:11,714+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d1102990>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:57:31,716+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ASBIN-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d11ab5f0>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:57:31,717+0000","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:161] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ADIXON-EQ&resolution=D&date_format=0&range_from=1705104000&range_to=1727222400&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7ec9d1102990>: Failed to resolve 'api-t1.fyers.in' ([Errno -3] Temporary failure in name resolution)\"))"},"timestamp":"2024-10-27 14:58:11,740+0000","service":"FyersAPI"}
